cmd/kubeadm/app/util/config/masterconfig.go

```diff
@@ -43,12 +43,34 @@ import (
 
 // SetInitDynamicDefaults checks and sets configuration values for the InitConfiguration object
 func SetInitDynamicDefaults(cfg *kubeadmapi.InitConfiguration) error {
+
+	if err := SetBootstrapTokensDynamicDefaults(&cfg.BootstrapTokens); err != nil {
+		return err
+	}
+
```

Comment:
just a nit: some of these empty new lines in the function can be removed.
please, live as is for now if you'd prefer.